### ISSUE TITLE
LWJGL2 OpenALMusic - ArrayIndexOutOfBoundsException on update()

### ISSUE THREAD

**Issue by someguy233** [libgdx/libgdx#5316]
#### Issue details
My game has a media player where you can seek randomly on an audio file. From player crash reports and feedback, it seems this happens if you repeatedly seek to near the end of the file.

#### Reproduction steps/code
Just the render() of an empty ApplicationListener:
```java
            Music music = null;

            @Override
            public void render() {
                int frameNumber = (int)Gdx.graphics.getFrameId();
                Gdx.app.log("CRASH", "Frame " + frameNumber);
                switch (frameNumber) {
                    case 0:
                        // Frame 1
                        music = Gdx.audio.newMusic(Gdx.files.external("any_audio_file.ogg"));
                        music.play();
                        break;

                    case 1:
                        // Frame 2
                        music.setPosition(26.300f);     // must be within at least 200ms from the end
                        break;

                    case 2:
                        // Frame 3
                        music.setPosition(26.300f);     // must be within at least 200ms from the end
                        break;

                        // Crash happens around frame 58 ~ 59
                }
            }
```

#### Version of LibGDX and/or relevant dependencies
1.9.9-SNAPSHOT

#### Stacktrace
```java
java.lang.ArrayIndexOutOfBoundsException: -1
	at com.badlogic.gdx.utils.FloatArray.pop(FloatArray.java:254)
	at com.badlogic.gdx.backends.lwjgl.audio.OpenALMusic.update(OpenALMusic.java:233)
	at com.badlogic.gdx.backends.lwjgl.audio.OpenALAudio.update(OpenALAudio.java:227)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:218)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126) 
```

#### Please select the affected platforms
- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [X] Windows
- [ ] Linux
- [ ] MacOS

**Comment by someguy233** [comment]
@mgsx-dev thanks! But I don't think it would fix my other issue #5318 . Maybe the problem originates from setPosition(), or perhaps the code that handles end of the stream buffers.

**Comment by mgsx-dev** [comment]
that's another issue (not related to this one). I'll take a look to the other one.
